phylogenetic/config/description.md

```diff
@@ -0,0 +1,29 @@
+We gratefully acknowledge the authors, originating and submitting
+laboratories of the genetic sequences and metadata for sharing their
+work. Please note that although data generators have generously shared
+data in an open fashion, that does not mean there should be free
+license to publish on this data. Data generators should be cited where
+possible and collaborations should be sought in some circumstances.
+Please try to avoid scooping someone else's work. Reach out if
+uncertain.
+
+#### Underlying data
+
+We curate sequence data and metadata from NCBI as starting point for
+our analyses. Curated sequences and metadata are available as flat
+files at:
+
+- [data.nextstrain.org/files/workflows/seasonal-cov/sequences.fasta.zst](https://.nextstrain.org/files/workflows/{pathogen}/sequences.fasta.zst)
+- [data.nextstrain.org/files/workflows/seasonal-cov/metadata.tsv.zs](https://data.nextstrain.org/files/workflows/seasonal-cov/metadata.tsv.zs)
```

Comment:
Aside from the URL edits, we are not automatically uploading these files to S3 yet. I don't think we want to advertise 404 links here.

Reply:
When we do upload these from ingest to S3, I also assume there will be metadata/sequences files _per virus_ instead of a single metadata/sequences files for seasonal cov.

Reply:
Thanks for the catch @joverlee521. I didn't realize the seasonal CoV curated data wasn't on S3. We can just pause on this PR until files exist.

Reply:
I've punted the URL addition out to https://github.com/nextstrain/seasonal-cov/issues/25, and will remove these lines from this PR, so this can get merged.